release-notes.txt

```diff
@@ -1,5 +1,70 @@
+Version 2.2.0 September 2017 PR: 1 - 169
+Open PR: 108, 145
+
+Grid:
+
+ - Extracted implementation ecl_nnc_geometry [1, 66, 75, 78, 80, 109].
+ - Fix bug in cell_contains for mirrored grid [51, 53].
+ - Extract subgrid from grid [56].
+ - Expose mapaxes [63, 64].
+ - grid.get_lgr - numbered lookup [83]
+ - Added NUMRES values to EGRID header [125].
+
+Build & testing:
+
+ - Removed warnings - added pylint [4, 5, 6, 10, 11, 12]
+ - Accept any Python 2.7.x version [17, 18]
+ - Remove ERT testing & building [3, 19]
+ - Changes to Python/cmake machinery [25, 30, 31, 32, 49, 52, 62].
+ - Added cmake config file [33, 44, 45, 47].
+ . Only *one* library [54, 55, 58, 69, 73, 77, 91, 133]
+ - Rmoved stale binaries [59].
+ - Require cmake >= 2.8.12 [67].
+ - Fix build on OSX [87, 88, 95, 103].
+ - Fix broken behavior with internal test data [97].
+ - Travis - compile with -Werror [122, 123, 127, 130]
+ - Started to support Python3 syntax [150, 161]
+ - Add support for paralell builds on Travis [149]
+
+C++:
+
+ - Removed use of deignated initializers [7].
+ - Memory leak in EclFilename.cpp [14].
+ - Guarantee C linkage for ecl_data_type [65].
+ - New smspec overload [89].
+ - Use -std=c++0x if -std=c++11 is unavailable [118]
+ - Make all of (previous( libutil compile with C++ [162]
+
+Well:
+
+ - Get well rates from restart files [8,20].
+ - Test if file exists before load [111].
+ - Fix some warnings [169]
+
+Core:
```

Comment:
Should we mention that we also now support MacOS, as all tests are passing? In addition the library compiles with `-werror=all` on Travis, meaning that we have fewer warnings.